### ISSUE TITLE
Feature request : Multiline text

### ISSUE THREAD

**Issue by lgersman** [natemoo-re/clack#25]
Your project looks promising !

Would you consider to add a Multiline Text Input like in Gum (https://github.com/charmbracelet/gum#write) ?

**Comment by natemoo-re** [comment]
Yep, I would love to add this! Looking at our current logic, most things are expecting a single line value, so this might take some internal refactoring.

**Comment by SebghatYusuf** [comment]
any updates here?

**Comment by deadcoder0904** [comment]
Would love this too bdw.

**Comment by natemoo-re** [comment]
🙈 better 1,170 days late than never—the multiline prompt has been released in [`@clack/prompts@1.3.0`](https://github.com/bombshell-dev/clack/releases/tag/%40clack%2Fprompts%401.3.0)
